src/mlpack/methods/ann/init_rules/nguyen_widrow_init.hpp

```diff
@@ -82,6 +82,22 @@ class NguyenWidrowInitialization
     W *= (beta / arma::norm(W));
   }
 
+  /**
+   * Initialize the elements of the specified weight matrix with the
+   * Nguyen-Widrow method.
+   *
+   * @param W Weight matrix to initialize.
+   */
+  template<typename eT>
+  void Initialize(arma::Mat<eT>& W)
+  {
+    RandomInitialization randomInit(lowerBound, upperBound);
+    randomInit.Initialize(W);
+
+    double beta = 0.7 * std::pow(W.n_cols, 1 / W.n_rows);
```

Comment:
That's an old error! `1 / W.n_rows` is an integer.
```suggestion
    double beta = 0.7 * std::pow(W.n_cols, 1.0 / W.n_rows);
```

There's the same error at line 81. I can't put a comment there since you didn't change the line. Could you fix it too?